_posts/2018-09-22-microservice-orchestration.markdown

```diff
@@ -866,7 +866,11 @@ Creating linkextractor_redis_1 ... done
 
 Now, that all three services are up, access the web interface by [clicking the Link Extractor](/){:data-term=".term1"}{:data-port="80"}.
 There should be no visual difference from the previous step.
-However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly.
+However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly. To check whether or not the Redis service is being used, we use `docker-compose exec` followed by the `redis` service name and the Redis CLI's [Monitor](https://redis.io/commands/monitor) command:
```

Comment:
```suggestion
However, if you extract links from a page with a lot of links, the first time it should take longer, but the successive attempts to the same page should return the response fairly quickly.
To check whether or not the Redis service is being utilized, we can use `docker-compose exec` followed by the `redis` service name and the Redis CLI's [Monitor](https://redis.io/commands/monitor) command:
```

I have followed one line per sentence convention when writing this Markdown file for better diffs. The above inline suggestion takes care of that and some minor changes in the sentence. You can either [incorporate these suggestions](https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request) or make necessary changes yourself.